packages/flutter/test/widgets/slivers_evil_test.dart

```diff
@@ -73,94 +73,99 @@ class TestViewportScrollPosition extends ScrollPositionWithSingleContext {
 void main() {
   testWidgets('Evil test of sliver features - 1', (WidgetTester tester) async {
     final GlobalKey centerKey = GlobalKey();
-    await tester.pumpWidget(Directionality(
-      textDirection: TextDirection.ltr,
-      child: ScrollConfiguration(
-        behavior: TestBehavior(),
-        child: Scrollbar(
-          child: Scrollable(
-            axisDirection: AxisDirection.down,
-            physics: const TestScrollPhysics(),
-            viewportBuilder: (BuildContext context, ViewportOffset offset) {
-              return Viewport(
+    await tester.pumpWidget(
+      MediaQuery(
```

Comment:
I'm hoping that the only change to this test is just adding the MediaQuery

Reply:
I think that's the case. Should I make the `MediaQuery` dependency optional?

Reply:
It's fine as is; just difficult to read github diffs sometimes.